kasenvr/project-athena
# Support blendshapes on model entities

Test plan:
- Run [this script](https://gist.githubusercontent.com/HifiExperiments/2fafca5d3bdd1e6368176cbd0bf4c6ad/raw/257d0f31268c0929edc01b62c61a3a8aea53feb6/BlendshapesTest.js)
- You'll see a woody in front of you.
- Press o/i/u and l/k/j to modify the entity's blendshapes:
![hifi-snap-by-SamGondelman-on-2020-01-05_02-27-03](https://user-images.githubusercontent.com/53453710/71778586-a7ff8f80-2f64-11ea-8e5f-1682233de7e4.gif)
